lib/index.js

```diff
@@ -377,6 +378,39 @@ module.exports = function (sdk, _) {
             this._obj = total; // @todo: do this the right way adhering to chai standards
         });
 
+        Assertion.addMethod('jsonSchema', function (schema, options) {
+            new Assertion(schema).to.be.an('object');
+
+            var ajv,
+                valid,
+                data;
+
+            options = Object.assign({
+                _parseJSON: true, // auto parse JSON by default
+                allErrors: true // check all rules collecting all errors
+            }, options);
+
+            // auto parse JSON if `json` methods exists
+            // this helps to chain with: pm.response.to.
+            if (options._parseJSON && this._obj && typeof this._obj.json === 'function') {
```

Comment:
Top level const.

Reply:
Strict check type as response. And rest treat as plain variables.